js/layers/help_layer/helpConfig.js

```diff
@@ -20,7 +20,7 @@ ubsApp.helpConfig={
                 "templateType":"static",
                 "id":"quizClose",
                 "style":"position:absolute;bottom:5%;left:90%;background-repeat: no-repeat;",
-                "src":"<button onclick=\"ubsApp.closeHelp();ubsApp.callServerClosePopup();\" style=\"padding:7px; color:white; font-weight:650; border:0; background:orangered;border-radius:5px; \">{{cancelTitle}}</button>"
+                "src":"<button onclick=\"ubsApp.socketCloseHelp();ubsApp.callServerClosePopup();\" style=\"padding:7px; color:white; font-weight:650; border:0; background:orangered;border-radius:5px; \">{{cancelTitle}}</button>"
```

Comment:
Can we have not change the callers and modify whatever is required inside closeHelp and startHelp pls